loader/generated/vk_loader_extensions.c

```diff
@@ -316,7 +316,7 @@ VKAPI_ATTR bool VKAPI_CALL loader_icd_init_entries(struct loader_icd_term *icd_t
 VKAPI_ATTR void VKAPI_CALL loader_init_device_dispatch_table(struct loader_dev_dispatch_table *dev_table, PFN_vkGetDeviceProcAddr gpa,
                                                              VkDevice dev) {
     VkLayerDispatchTable *table = &dev_table->core_dispatch;
-    assert(table->magic == DEVICE_DISP_TABLE_MAGIC_NUMBER);
+    if (table->magic != DEVICE_DISP_TABLE_MAGIC_NUMBER) { abort(); }
```

Comment:
@MarkY-LunarG This is the only change that I'm not confident in. This should be fine since loader_init_device_dispatch_table is called once during setup and should *never* get clobbered and have a functioning application. But I only made it abort because the death test wouldn't pass with just an assert (release builds wouldn't crash).

Reply:
I think it should be fine as well.